### ISSUE TITLE
What is the location format?

### ISSUE THREAD

**Issue by importantlyearnest** [Jacopo1891/MMM-GoogleTrafficTimes#9]
Apologies, I don't understand the location (destination) format. It's a structure I've never seen. I'm in the US.

Would I just use lat, long? What does the format for that look like?

**Comment by Jacopo1891** [comment]
Hi! For now, the destination address to set is exactly what you would search for on Google Maps to go to a place – I mean the street name, house number, and city. 

(For places marked as locations on Google Maps like gyms, superstors etc, just the name of the place and the city should be enough; for example, with 'Empire State Building, New York' It should be able to calculate the route correctly.)

I'm pretty sure it can also handle latitude and longitude, but I've never tried it.

Update:
I also tried with latitude and longitude, and it works. 
For example, the coordinates of the Empire State Building from Google Maps are '40,7478193, -73,9850879' (probably because I have Italian language). 
To make them work, I replaced the commas with periods and removed the space after the comma, like this: '40.7478193,-73.9850879', and it works.
